src/main/python/pybuilder/plugins/python/sphinx_plugin.py

```diff
@@ -62,24 +99,41 @@ def assert_sphinx_is_available(logger):
 def sphinx_generate(project, logger):
     """Runs sphinx-build against rst sources for the given project.
     """
-    logger.info("Running sphinx-build")
-
-    log_file = project.expand_path("$dir_target/reports/{0}".format("sphinx-build"))
+    task_name = getattr(sphinx_generate, NAME_ATTRIBUTE)
```

Comment:
In retrospect I think it would be okay to have "sphinx_generate" as a string here, to avoid the coupling with the name attribute (or tasks could have a public way of figuring out their name).
OTOH I also like the approach you took here because this would survive automated refactorings (or replacing the `name` argument in the decorator)